CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ set(BUILD_SERVER FALSE CACHE BOOL "Build server")
 set(BUILD_UNITTESTS TRUE CACHE BOOL "Build unittests")
 set(BUILD_BENCHMARKS FALSE CACHE BOOL "Build benchmarks")
 set(BUILD_DOCUMENTATION TRUE CACHE BOOL "Build documentation")
+set(BUILD_UI FALSE CACHE BOOL "Build experimental UI API")
```

Comment:
Please specify that `Build Client` and `USE_SDL2` are required here, rather than just fatal erroring when they try it.